tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -6403,13 +6405,17 @@ <h2>Show similarity to selected datapoint</h2>
           let attributionColorRange = [];
           if (this.minAttribution < 0) {
             attributionDomain.push(this.minAttribution);
```

Comment:
This looks good. However, since we have so many color levels now, I wonder if it is better to pass an object with keys as the color strings and values as the domain values above if this is the only place they are being used.

Reply:
makes sense as a future improvement